### PR TITLE
Initial version of the site

### PR DESCRIPTION
**Pull request by mahtabsabet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,41 @@
...
+## Contributing
+
+Please check out the CONTRIBUTING doc for tips on making a successful contribution.
+
+## How it works
+
+At this stage, this repo is merely a prototype for the real ShipNoise.Net, a proof-of-concept for future development. 
+It is automatically deployed to the AWS Amplify environment from this repo's main branch. Pushes to the main branch trigger a new deployment.
+
+The demo site is available at: https://main.d27wklsx9a9q36.amplifyapp.com/
```
Comment (scottveirs) [inline README.md:16]: This link is given just above, so probably doesn't need to be repeated here... 
(This is mostly Scott just practicing leaving a single-line comment.)

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+var Component = React.Component;
+ 
+export default class App extends Component {
+  render() {
+    return (
+      <div>
+      <h1>ShipNoise.net</h1>
+      <h3>Characterizing ship noise in orca habitats</h3>
+      
+      <p>Ship noise dominates the acoustic habitat of the Southern Resident Orca Whales within the Salish Sea.</p>
```
Comment (scottveirs) [inline src/App.js:14]: This should probably use the more (scientifically) conventional "Southern Resident Killer Whales" and leave "orcas" for the next line. That will be good for SEO and the two lines together will "speak" to biological experts and citizen scientists, alike.

File: src/App.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      <p>What did the orca whales hear today in Puget Sound?</p>
```
Comment (scottveirs) [inline src/App.js:15]: Switch from "orca whales" to the more commonly used "orcas" ?

**Review by ivanoats — CHANGES_REQUESTED** [review]
just update src/App.test.js and you're good to go 👍🏼

**Pull request merged**